importer/requirements.txt

```diff
@@ -8,6 +8,7 @@ pytest>=7.4.2
 jsonschema==4.21.1
 mock==5.1.0
 python-magic==0.4.27
+python-magic-bin==0.4.14
```

Comment:
Hi @muhammad-levi,
technically this is only required for windows. So maybe add it to the end of the file with an environment marker eg
```

# Windows requirements
python-magic-bin==0.4.14; sys_platform == 'win32'
```
or add a separate `requirements-windows.txt` file